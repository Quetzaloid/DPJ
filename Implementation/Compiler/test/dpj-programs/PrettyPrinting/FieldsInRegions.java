public class FieldsInRegions {
    public region r1, r2, r3;
    public int a in Root:FieldsInRegions.r1, b = 17, c in Root:FieldsInRegions.r2;
    public int a_short in r1, b_short = 17, c_short in r2;
    private float d in Root:FieldsInRegions.r2:FieldsInRegions.r3;
    private float d_short in r3;
    protected FieldsInRegions e in Root:FieldsInRegions.r2:FieldsInRegions.r3 = null;
    protected FieldsInRegions e_short in r3 = null;
}
