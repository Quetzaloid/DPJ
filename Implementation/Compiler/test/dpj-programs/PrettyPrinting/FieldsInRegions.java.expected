
public class FieldsInRegions {
    
    public FieldsInRegions() {
        super();
    }
    public region r1;
    public region r2;
    public region r3;
    public int a in Root:FieldsInRegions.r1;
    public int b = 17;
    public int c in Root:FieldsInRegions.r2;
    public int a_short in r1;
    public int b_short = 17;
    public int c_short in r2;
    private float d in Root:FieldsInRegions.r2:FieldsInRegions.r3;
    private float d_short in r3;
    protected FieldsInRegions e in Root:FieldsInRegions.r2:FieldsInRegions.r3 = null;
    protected FieldsInRegions e_short in r3 = null;
}
